### ISSUE TITLE
More explanation during installation needed

### ISSUE THREAD

**Issue by jkpubsrc** [MarketSquare/robotframework-browser#953]
I just installed this package as described and run `rfbrowser init` as expected.
After I just gave it a brief look. And was quite surprised:
`rfbrowser init` installed 850 MByte of data.

I'm pretty sure there is a good reason for it. But unfortunately I don't find any information about these reasons. In my opinion if a piece of software installs over 850 MByte it would be quite reasonable to give at least some information why *that* amount of data is necessary, and at least a hint that *that* amount of data will be installed.

Please update the documentation and provide more information about what is going on here.

Thank you.

**Comment by aaltat** [comment]
Yes, there is a reason: Browser library is based on playwright and playwright comes with browser included. Meaning that when playwright installed, it will by default install also Chromium, Firefox and WebKit browsers (even users would have those browser installed.) It is possible to skip browser installation, by using `rfbrowser init --skip-browsers`. Also playwright supports more fine-grained installation options, but we do not yet support those.

**Comment by aaltat** [comment]
@all-contributors please add @jkpubsrc  for bugs
